VirtoCommerce.Storefront/DependencyInjection/ServiceCollectionExtension.cs

```diff
@@ -111,6 +111,7 @@ public static void AddPlatformEndpoint(this IServiceCollection services, Action<
             services.AddAutoRestClient((credentials, httpClient, disposeHttpClient, baseUri) => new CatalogModuleClient(credentials, httpClient, disposeHttpClient) { BaseUri = baseUri });
             services.AddSingleton<ICatalogModuleCategories>(sp => new CatalogModuleCategories(sp.GetRequiredService<CatalogModuleClient>()));
             services.AddSingleton<ICatalogModuleProducts>(sp => new CatalogModuleProducts(sp.GetRequiredService<CatalogModuleClient>()));
+            services.AddSingleton<ICatalogModuleAssociations>(sp => new CatalogModuleAssociations(sp.GetRequiredService<CatalogModuleClient>()));
```

Comment:
Are you sure that this related to this PR?

Reply:
Yes, it does. It's just actualization of Catalog module controllers changes.